### DISCUSSION TITLE
First 30 seconds not transcribed correctly

### DISCUSSION THREAD

**Discussion by Abdurrafey-Siddiqui** [openai/whisper#704] [Q&A]
In the first 30 seconds of the audio, the first phrase of the speaker is transcribed but then that phrase remains all the way until 30 seconds finish even though the speaker is saying other things during that time frame. Then it gets fixed and works correctly for the rest of the audio. How to fix this?

**Comment by devvythelopper** [comment]
workaround: add an empty 30 second long sequence before the audio input, then shift all the times of in the SRT by 30 seconds toward the beginning. If I come around to write a script for it, I'll link it
